documentation/modules/migration-plan-options-ui.adoc

```diff
@@ -8,30 +8,32 @@
 
 On the *Plans for virtualization* page of the {ocp} web console, you can click the {kebab} beside a migration plan to access the following options:
 
-* *Get logs*: Retrieves the logs of a migration. When you click *Get logs*, a confirmation window opens. After you click *Get logs* in the window, wait until *Get logs* changes to *Download logs* and then click the button to download the logs.
-* *Edit*: Edit the details of a migration plan. You cannot edit a migration plan while it is running or after it has completed successfully.
-* *Duplicate*: Create a new migration plan with the same virtual machines (VMs), parameters, mappings, and hooks as an existing plan. You can use this feature for the following tasks:
+* *Edit Plan*: Edit the details of a migration plan. You cannot edit a migration plan while it is running, or after it has been completed successfully.
```

Comment:
"Edit Plan"  - not sure if it was ever implemented like this,  but for now this is not accurate since this Edit plan option within the menu is always enabled once the plan was created.
That's the standard for the console and once clicked, you are referred to plan details page. 

What's correct is that if the plan is still running/completed successfully, then the following specific plan's properties should not be editable, only view mode:

1. All properties under the plan details tab -> `Settings` section (e.g.  warm/cold migration, Target namespace, Disk decryption passphrases,  Transfer Network , Preserve static IPs, Preserve CPU model etc...)
2. . The plan's mapping under the `Mappings` tab
4.  The hooks under the `Hooks` tab.

Note that removing VMs for a running plan under the `Virtual Machines'` tab -  is still allowed.